src/wazuh_modules/vulnerability_scanner/tests/databaseFeedManager/databaseFeedManager_test.cpp

```diff
@@ -134,6 +136,58 @@ void DatabaseFeedManagerTest::SetUp()
         rocksDBWrapper.put(PACKAGE_NAME + "_" + item.at("candidates").at(0).at("cveId").get<std::string>(),
                            vulnerabilityCandidate);
     }
+
+    nlohmann::json jsonFileContent;
+    nlohmann::json jsonItem = R"(
+        {
+            "offset": 0,
+            "type": "create",
+            "version": 1,
+            "context": "all_vendors_071123",
+            "resource": "CVE-0000-0000",
+            "payload": {
+                "containers": {
+                    "cna": {
+                        "providerMetadata": {
+                            "dateUpdated": "2017-05-11T14:29:20Z",
+                            "orgId": "00000000-0000-4000-A000-000000000003",
+                            "shortName": "nvd"
+                        },
+                        "rejectedReasons": [
+                            {
+                                "lang": "en",
+                                "value": "** REJECT **"
+                            }
+                        ]
+                    }
+                },
+                "cveMetadata": {
+                    "assignerOrgId": "00000000-0000-4000-A000-000000000003",
+                    "assignerShortName": "nvd",
+                    "cveId": "CVE-0000-0000",
+                    "datePublished": "2017-05-11T14:29:20Z",
+                    "dateUpdated": "2017-05-11T14:29:20Z",
+                    "state": "REJECTED"
+                },
+                "dataType": "CVE_RECORD",
+                "dataVersion": "5.0"
+            }
+        }
+    )"_json;
+    int cveNumber = 1;
+    for (auto itemNumber = 0; itemNumber < 1000; itemNumber++)
```

Comment:
We are creating this file many times, once per test, but using it only once. This makes the tests take longer.
We could create it only for the test that requires it.

Reply:
fixed